.github/workflows/ci.yml

```diff
@@ -23,8 +23,8 @@ jobs:
         uses: volta-cli/action@v4
       - name: Install Dependencies
         run: yarn install --frozen-lockfile
-      - name: Precompile Typescript
-        run: yarn workspace ember-sinon-qunit run prepack
+      - name: Build v2 Addon
+        run: yarn workspace ember-sinon-qunit run build
```

Comment:
maybe we can rely on `prepare` script like it's done in blueprint https://github.com/embroider-build/addon-blueprint/blob/main/src/root-package-json.js#L52 ?